src/main/java/RandomComputerPlayer.java

```diff
@@ -0,0 +1,35 @@
+import java.util.List;
+import java.util.Random;
+
+public class RandomComputerPlayer implements Player{
+
+    private String marker = "";
+    private String name = "Computer";
+
+    public String getMarker(){
+        return marker;
+    }
+
+    public String getName(){
```

Comment:
should getName and getMarker also be on the Player interface? Are they part of the contract that players need to fufill?